lib/apis/query.ts

```diff
@@ -172,6 +172,11 @@ export class Query {
     return await this.postRequest<string>(this.api, 'getNonce', { accountId: accountAddress, nonceType });
   }
 
+  async getAnchorNonce(chainId: number): Promise<string> {
+    const parsedChainId = chainId === 0 ? `${chainId}`: chainId
```

Comment:
why don't we convert chainId to string for all cases?

Reply:
good catch will adjust